bucket/seafile-syncing-client.json

```diff
@@ -0,0 +1,27 @@
+{
+    "homepage":  "https://www.seafile.com/",
+    "description": "Seafile desktop syncing client",
+    "version":  "7.0.9",
+    "license":  "Apache-2.0",
+    "url":  "https://download.seadrive.org/seafile-7.0.9-en.msi",
+    "hash":  "3f837823c23403ad2d5aec29f81028aaf1e86055e2fa45a8b67fcf94f59ddade",
+    "extract_dir":  "PFiles\\Seafile\\bin",
+    "post_install": "[Environment]::SetEnvironmentVariable(\"SEAFILE_DATA_DIR\",\"$DIR\\data\",\"User\")",
+    "uninstaller": {
+        "script": "[Environment]::SetEnvironmentVariable(\"SEAFILE_DATA_DIR\",$null,\"User\")"
+    },
+    "persist": "data",
+    "shortcuts": [
+        [
+            "seafile-applet.exe",
+            "Seafile Desktop Syncing Client"
+        ]
+    ],
+    "checkver": {
+        "url": "https://www.seafile.com/en/download/",
```

Comment:
Use changelog https://download.seafile.com/published/seafile-manual/changelog/client-changelog.md

Reply:
Is `checkver` rendering the page? If not, the changelog URL itself won't help.

Should I use https://download.seafile.com/api/v2.1/wikis/seafile-manual/content/?p=%2Fchangelog%2Fclient-changelog.md&format=json instead with `(\d+\.\d+\.\d+)` as the regex?